### DISCUSSION TITLE
Welcome to Labs-Windows Discussions!

### DISCUSSION THREAD

**Discussion by michael-hawker** [CommunityToolkit/Labs-Windows#94] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with the members of our community.

Discussions here in 🧪 Toolkit Labs for Windows 💻 will be the starting point for any new feature development of the Windows Community Toolkit. If you have an idea for a new feature, please post a new 💡 Idea!

You can also:
  * Ask questions about experiments or labs itself.
  * Share new ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kvaibhav-2005** [comment]
hey hello is there a way to boot windows into different spaces using different passwords but same account. like if I'm forced to open my laptop in front of people i want to be able to write in my second password and open up a different account-ish place kinda like a safe space

**Reply by michael-hawker** [reply]
@kvaibhav-2005 this is a developer control library GitHub not related directly to the Windows OS and its features, so we can't answer your question here. Though to my knowledge, there's not a way to do what you're asking. You can create an alternate profile/account, but not tied directly to the same account with a different password.

**Comment by JoeTomkinson** [comment]
Morning all, I wasn't sure where the best place to ask was, but I've been following the markdown work in labs pretty closely and I had a couple of questions.

The first was if there's an ETA for it to be moved to production release?

The second was is there a trick to opening the project in VS22, for some reason it want's to 'update' it but doesn't really specify why and then fails with an exception. I just really want to get hands on and take a look at how it works in reality.
